graph/resolver/post.resolvers.go

```diff
@@ -64,6 +66,45 @@ func (r *postResolver) UpdatedAt(ctx context.Context, obj *model.Post) (string,
 	return obj.UpdatedAt.Format(time.RFC3339), nil
 }
 
+func (r *postResolver) MentionedEntities(ctx context.Context, obj *model.Post) ([]model.Entity, error) {
+	var entities []model.Entity
```

Comment:
hmm, is there a better way to fetch this?

Reply:
My suggestion would be to batch up the types and do 2 total fetches (GetParticipantsByIDs; GetDiscussionsByIDs)